TombEngine/Game/itemdata/creature_info.h

```diff
@@ -71,6 +71,60 @@ struct LOTInfo
 	bool CanMonkey	  = false;
 };
 
+struct CreatureBiteInfo
```

Comment:
Since we're renaming this, why don't we come up with a name better than "bite"? :P